src/NServiceBus.AcceptanceTests/Routing/When_publishing_an_interface.cs

```diff
@@ -42,7 +43,8 @@ class Context : ScenarioContext
             public Type EventTypePassedToRouting { get; set; }
         }
 
-        class Publisher : EndpointConfigurationBuilder
+        [SuppressMessage("ReSharper", "MemberCanBePrivate.Global")]
```

Comment:
Since andreas is investigating a better overall solution, I'd say we treat this as a pain killer for now and also leave out the attribute as we don't use it anywhere else in the tests.